app/src/app/components/header/HeaderMenu.tsx

```diff
@@ -0,0 +1,27 @@
+import { Menu } from "lucide-react";
+import { NavLink } from "react-router-dom";
+import { Button } from "../Base/Button";
+import { DropdownMenu, DropdownMenuContent, DropdownMenuItem, DropdownMenuTrigger } from "../Base/DropdownMenu";
+import { LeftNavItems } from "./LeftNav";
+
+export const HeaderMenu = () => {
+  return (
+    <DropdownMenu>
+      <DropdownMenuTrigger>
+        <Button variant="ghost" size="icon">
+          <Menu />
+        </Button>
+      </DropdownMenuTrigger>
+      <DropdownMenuContent className="w-48">
+        {Object.entries(LeftNavItems).map(([to, title]) => (
+          <DropdownMenuItem key={to} asChild>
+            <NavLink to={to}>{title}</NavLink>
+          </DropdownMenuItem>
+        ))}
+        <DropdownMenuItem asChild>
+          <NavLink to="/accessibility">Accessability</NavLink>
```

Comment:
Is `HeaderMenu` only visible on smaller screens? And it's `LeftNav` you get on larger ones? Could that be made clearer in a comment or by component name?

Sp on Accessibility again.. 

I think we need to exclude the LeftNav items if not authenticated..?

Reply:
good call sorted